src/CBot/CBotEnums.h

```diff
@@ -245,16 +248,15 @@ enum CBotError : int
     CBotErrNoRetVal      = 6003, //!< function did not return results
     CBotErrNoRun         = 6004, //!< Run() without active function
     CBotErrUndefFunc     = 6005, //!< calling a function that no longer exists
-    CBotErrNotClass      = 6006, //!< this class does not exist
```

Comment:
CBotErrNotClass is compile error (no runtime) and has to have value less than 6000, otherwise tests won't work correct. However, I will preserve other error values (between 6005 and 6007 will be gap).